src/collections/operations/updateByID.ts

```diff
@@ -98,26 +95,24 @@ async function updateByID<TSlug extends keyof GeneratedTypes['collections']>(
   // Retrieve document
   // /////////////////////////////////////
 
-  const query = await Model.buildQuery({
+
+  const findArgs: FindArgs = {
+    collection: collectionConfig.slug,
     where: combineQueries({ id: { equals: id } }, accessResults),
-    payload,
-    locale,
-  });
+    locale: req.locale,
```

Comment:
`locale` is deconstructed to it's own const above line can be changed to just `locale,`